ui/src/services/api.js

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+import { Config, Core_v1Api } from '@kubernetes/client-node';
+
+let config, coreV1;
```

Comment:
Please have a ticket to get rid of global state, preferably...

Reply:
I do not know how to get rid of them? :)

Reply:
Construct them at some kind of entry-point of your application, thread them through, and pass when invoking e.g. `getNodes`.